src/advanced/consensus/specification.md

```diff
@@ -351,7 +351,10 @@ round are placed into a separate queue (`queued`).
 
 - If the timeout does not match the current height and round, skip further
   timeout processing.
-- Add a timeout (its length is specified by `round_timeout`) for the next round.
+- Add a timeout (its length is specified by the following formula
+  `first_round_timeout + (N-1)*round_timeout_increase`) for the Nth round.
```

Comment:
For additional clarity, I'd suggest to reorganize the formula as

```none
first_round_timeout * (1 + (N - 1) * q))
```

where `q = 0.1`, if I understand correctly.